packages/cli/src/formatters/json.ts

```diff
@@ -1,6 +1,21 @@
-import { Formatter } from './types';
+import { ISpectralDiagnostic } from '@stoplight/spectral-core';
+import { Formatter, FormatterOptions } from './types';
 
-export const json: Formatter = results => {
+import { groupBySource, uniqueErrors, getCountsBySeverity, getScoringText } from './utils';
+
+const version = process.env.npm_package_version;
+
+export const json: Formatter = (results: ISpectralDiagnostic[], options: FormatterOptions) => {
+  let spectralVersion = '';
+  let groupedResults;
+  let scoringText = '';
+  if (options.scoringConfig !== void 0) {
+    if (options.scoringConfig.customScoring !== undefined) {
+      spectralVersion = `${options.scoringConfig.customScoring} ${version as string}`;
```

Comment:
I don't think including a version makes much sense.
The result does not directly depend on the version of the CLI client, but on the ruleset you use, so seems like we could just not include it?